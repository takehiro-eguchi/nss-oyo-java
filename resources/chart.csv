130.0,30.0,131.0,31.0
20,1,130.1,30.1
20,2,130.1,30.2
20,3,130.2,30.2
20,4,130.2,30.1
20,5,130.2,30.3
20,6,130.2,30.4
20,7,130.5,30.4
20,8,130.5,30.5
20,9,130.5,30.6
20,10,130.2,30.6
20,11,130.2,30.8
20,12,130.4,30.8
20,13,130.4,31.0
20,14,130.6,31.0
20,15,130.6,30.5
20,16,130.9,30.1
25,17,1,2,3,4,1
25,18,5,6,7
25,19,8,9,10,11,12,13
25,20,13,14,15,8
30,21,17
30,22,19,20
40,1,■
40,2,◎
40,3,＝
40,4,×
10,1,島,21,1,100
10,2,陸地,22,1,100
10,3,灯台,3,2,150
10,4,灯台,9,2,150
10,5,海底ケーブル,18,3,100
10,6,障害物,16,4,200