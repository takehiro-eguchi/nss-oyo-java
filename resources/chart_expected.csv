     ＿＿＿＿＿＿＿＿＿＿＿
31.0|　　　　■■■　　　　｜
    |　　　　■■■　　　　｜
    |　　■■■■■　　　　｜
    |　　■■■■■　　　　｜
    |　　■■■◎■　　　　｜
    |　　　　　■■　　　　｜
    |　　＝＝＝＝　　　　　｜
    |　　＝　　　　　　　　｜
    |　■◎　　　　　　　　｜
    |　■■　　　　　　×　｜
30.0|　　　　　　　　　　　｜
　　 －－－－－－－－－－－
     130.0             131.0